src/Pages/Home/Home.module.scss

```diff
@@ -4,4 +4,12 @@
   h1 {
     font-weight: bold;
   }
+  .articlesection {
```

Comment:
Always add an empty line to separate two style blocks (add this change wherever necessary)